SECURITY.md

```diff
@@ -0,0 +1,31 @@
+# Security Policy
+
+1. [Reporting security problems](#reporting)
+1. [Security Bug Bounties](#security-bug-bounties)
+1. [Scope](#scope)
+
+## Reporting
+
+**DO NOT CREATE A GITHUB ISSUE** to report a security problem.
+
+Instead please use this [Report a Vulnerability](https://github.com/mithraiclabs/spl-token-staking/security/advisories/new) link.
+Provide a helpful title and detailed description of the problem.
+
+If you haven't done so already, please **enable two-factor auth** in your GitHub account.
+
+Expect a response as fast as possible in the advisory, typically within 72 hours.
+
+--
+
+If you do not receive a response in the advisory, send an email to
+security@psyoptions.io with the full URL of the advisory you have created. DO NOT
+include attachments or provide detail sufficient for exploitation regarding the
+security issue in this email. **Only provide such details in the advisory**.
+
+## Security Bug Bounties
+
+TBD ... todo :)
```

Comment:
Not much point without a cash value! Might as well say 1-10k at our discretion if we want a non commitment here.

Reply:
We will link to the Armada doc's site once we have bug bounty info up there